nvflare/fuel/hci/client/cli.py

```diff
@@ -79,6 +84,8 @@ def __init__(
         credential_type: str = CredentialType.PASSWORD,
         cmd_modules: Optional[List] = None,
         debug: bool = False,
+        cli_history_dir: str = "",
+        cli_history_size: int = 1000,
     ):
```

Comment:
Can you move the docstring from class level under `__init__` and add the new arguments?
Thanks!

Reply:
Done